src/engine/renderer/tr_shade.cpp

```diff
@@ -3143,26 +3143,26 @@ void Tess_End()
 	}
```

Comment:
why is it safe to return early here but now below?

Reply:
The most important thing is to zero out the counters: numIndexes etc.